packages/quantic/force-app/main/default/lwc/quanticQuickviewContent/quanticQuickviewContent.js

```diff
@@ -0,0 +1,60 @@
+import { LightningElement, api } from 'lwc';
+// @ts-ignore
+import defaultTemplate from './quanticQuickviewDefault.html';
+// @ts-ignore
+import youtubeTemplate from './quanticQuickviewYoutube.html';
+
+
+/** @typedef {import("coveo").Result} Result */
+
+const allowedHost = {youtube: 'https://youtube.com/'};
+
+/**
+ * The `QuanticQuickviewContent` component renders a content template based on the type of the content (Youtube, other).
+ * @category Result Template
+ * @example
+ * <c-quantic-quickview-content result={result} content-url={contentURL}></c-quantic-quickview-content>
+ * @internal
```

Comment:
While this is actually pretty cool and the doc generator could be configured to use a tag like this instead, currently the way to omit a component from the reference doc JSON is to set `@category Internal`

Reply:
You can remove this now that the @category has been updated.